hibernate-core/src/main/resources/org/hibernate/hibernate-mapping-4.0.xsd

```diff
@@ -1396,7 +1396,7 @@ arbitrary number of queries, and import declarations of arbitrary classes.
     </xs:sequence>
     <xs:attribute name="cache-mode" type="cache-mode-attribute"/>
     <xs:attribute name="cache-region" type="xs:string"/>
-    <xs:attribute name="cacheable" default="false" type="xs:boolean"/>
```

Comment:
Is this change really necessary?

Reply:
it seems so

Reply:
without we have test failures

Reply:
Because this change is actually a schema change for hbm.xml which we shouldn't do lightly.